waiter/integration/waiter/token_request_test.clj

```diff
@@ -82,33 +103,29 @@
           num-tokens-to-create 10
           tokens-to-create (map #(str "token" %1 "." token-prefix) (range num-tokens-to-create))
           current-user (System/getProperty "user.name")]
+
       (log/info "creating the tokens")
       (doseq [token tokens-to-create]
-        (let [{:keys [body status]} (post-token waiter-url {:health-check-url "/custom-endpoint"
-                                                            :token token
-                                                            :name service-id-prefix})]
-          (is (= 200 status) (str "Error: " body))))
+        (let [response (post-token waiter-url {:health-check-url "/probe"
+                                               :name service-id-prefix
+                                               :token token})]
+          (assert-response-status response 200)))
+
       (log/info "ensuring tokens can be retrieved and listed on each router")
       (doseq [token tokens-to-create]
         (doseq [[_ router-url] (routers waiter-url)]
-          (let [{:keys [body] :as token-response} (get-token router-url token :cookies cookies)
-                token-description (try (json/read-str (str body))
-                                       (catch Exception _
-                                         (is false (str "Failed to parse token " body))))]
-            (assert-response-status token-response 200)
-            (is (= "/custom-endpoint" (token-description "health-check-url")))
-            (is (= service-id-prefix (token-description "name")))
-            (is current-user (token-description "owner"))
-            (is (not-any? #(contains? token-description %) ["cmd" "cpus" "mem" "run-as-user" "permitted-user"])
-                (str "Unexpected content in token response: " body)))
+          (let [response (get-token router-url token :cookies cookies)]
+            (assert-token-response response service-id-prefix false))
           (let [{:keys [body] :as tokens-response} (list-tokens router-url current-user :cookies cookies)
                 tokens (json/read-str body)]
             (assert-response-status tokens-response 200)
             (is (some (fn [token-entry] (= token (get token-entry "token"))) tokens)))))
-      (log/info "deleting the tokens")
+
+      (log/info "deleting the tokens (not erasing)")
```

Comment:
Here we use "erase". It's inconsistent, but I think it's more clear than "excise"